jbpm-flow/src/main/java/org/jbpm/workflow/instance/impl/NodeInstanceImpl.java

```diff
@@ -78,12 +82,12 @@
     protected int slaCompliance = ProcessInstance.SLA_NA;
     protected Date slaDueDate;
     protected long slaTimerId = -1;
-    protected boolean aborted = false;
```

Comment:
Node instances are stored as part of the binary state using protobuf, wouldn't we have to keep the aborted field to keep backwards compatibility?  Would we need to generate the protobuf files?  From what I understand this field is just used to pass information around (to the listener generating the logs), so I guess it probably isn't going to be necessary to be serialized (actually aborted node instances should not be serialized), but unless this aborted field was somehow excluded in protobuf marshalling, and/or cancelType will be too?

Reply:
made it transient.